packages/cli/src/commands/test.js

```diff
@@ -86,22 +110,22 @@ export const handler = async ({
   }
 
   if (sides.includes('api')) {
-    args.push('--runInBand')
+    jestArgs.push('--runInBand')
   }
 
-  args.push(
+  jestArgs.push(
     '--config',
     `"${require.resolve('@redwoodjs/core/config/jest.config.js')}"`
   )
 
   if (sides.length > 0) {
-    args.push('--projects', ...sides)
+    jestArgs.push('--projects', ...sides)
   }
 
-  try {
-    const cacheDirDb = `file:${ensurePosixPath(CACHE_DIR)}/test.db`
-    const DATABASE_URL = process.env.TEST_DATABASE_URL || cacheDirDb
+  const cacheDirDb = `file:${ensurePosixPath(CACHE_DIR)}/test.db`
+  const DATABASE_URL = process.env.TEST_DATABASE_URL || cacheDirDb
 
+  try {
     // Create a test database
     if (sides.includes('api')) {
       await execa(
```

Comment:
Maybe this code is easier to read when it's adjacent to where it's used, it also only needs to be declared when it's an api side.
```suggestion
      const cacheDirDb = `file:${ensurePosixPath(CACHE_DIR)}/test.db`
      const DATABASE_URL = process.env.TEST_DATABASE_URL || cacheDirDb
      await execa(
```